### ISSUE TITLE
PHPUnit test require real values in .env file

### ISSUE THREAD

**Issue by donkidd** [lesstif/php-jira-rest-client#189]
@lesstif I was looking into adding functionality to this repo related to Boards and Sprints but when I initially download the repo and try to run PHPUnit, why does PHPUnit require a real server to connect to?  Shouldn't the tests just be calling a mocked Jira server instead of a real one?

**Comment by donkidd** [comment]
The problem is that I don't have a real test jira server that I can use to run tests.  And even if I did I would have to create certain issues in my server so that the tests would pass.

**Comment by lesstif** [comment]
i think that code contributor should have a  real jira server for testing, but if you have not it you could skip unittest on real server.

```sh
phpunit tests/MapperTest.php
phpunit tests/AssigneeTest.php
```

**Comment by donkidd** [comment]
I would think that if you are actually doing a unit Test you shouldn't really be touching other then the Unit of code that you are working on.  If you are testing the actual responses you are getting from a server that isn't exactly a unit test.

**Comment by lesstif** [comment]
ok, i agree with you.
exactly, it 's not unit test, it is more like an integration test.

but i don't want to modify current status, because i have no time .

**Comment by donkidd** [comment]
I think I have unit tests working on atleast my recent changes.  I updated sprint service after fixing it, so it can be unit tested.  I also created a board service which is also unit testable.  You want to check it out, let me know what you think. I think the tests can be improved but they are a start.  I had to change the way the service classes worked a little so I haven't spent time to go redoing the others, but let me know.  I have it in my [Forked Repo](https://github.com/donkidd/php-jira-rest-client/tree/%2391-Start-Pulling-VelocityBySprint).  I can submit a PR if you prefer, let me know.
